x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_logs/agent_logs.tsx

```diff
@@ -195,6 +197,31 @@ export const AgentLogsUI: React.FunctionComponent<AgentLogsProps> = memo(({ agen
     return semverGte(agentVersionWithPrerelease, '7.11.0');
   }, [agentVersion]);
 
+  if (!isLogFeatureAvailable) {
+    return (
+      <EuiCallOut
+        size="m"
+        color="warning"
+        title={
+          <FormattedMessage
+            id="xpack.fleet.agentLogs.oldAgentWarningTitle"
+            defaultMessage="The logs viewer is only compatible with Elastic Agents version 7.11 and higher. Please upgrade your Elastic Agent on the Agents details page, or by {downloadLink} a newer version."
```

Comment:
```suggestion
            defaultMessage="The logs viewer requires Elastic Agent 7.11 or higher. To upgrade an agent, go to the Agents page, or {downloadLink} a newer version."
```

Reply:
Does the page actually say "Agents details"? If not, use my edit above ^^ and if possible make the text a link to the page.

If it does say Agents details, it sounds odd and needs work. (My apologies for not spinning up the latest build.)

Reply:
Thanks! And yes it should say "To upgrade an agent, go to the Agents page"

Reply:
@dedemorton For consistency, does it work to use "Logs view" instead of "logs viewer." ?

> The Logs view requires Elastic Agent 7.11 or higher. To upgrade an agent, go to the Actions menu, or {downloadLink} a newer version.

Reply:
Yes, that's better @gchaps!